code/core/src/manager/components/sidebar/LegacyRender.tsx

```diff
@@ -5,23 +5,23 @@ import type { TestProviders } from 'storybook/internal/core-events';
 import { useStorybookApi } from 'storybook/internal/manager-api';
```

Comment:
Do you agree that we should get rid of this LegacyRender for 9.0, and migrate VTA to the  modern one?

Reply:
When we update VTA to use UniversalStore, it's going to have to use a Storybook version that supports it. Once we do that, we can update VTA to provide its own rendering and get rid of LegacyRender.